### ISSUE TITLE
Webpack compatibility

### ISSUE THREAD

**Issue by captnCC** [alexa/alexa-skills-kit-sdk-for-nodejs#375]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ x ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

Webpack gives a warning while bundling a Typescript file importing from the Alexa SDK
 
## Expected Behavior

Webpack runs without warning 

## Current Behavior

Webpack gives a warning:
<pre>
WARNING in ./node_modules/ask-sdk-core/dist/util/AskSdkUtils.js
34:22-67 Critical dependency: the request of a dependency is an expression
 @ ./node_modules/ask-sdk-core/dist/util/AskSdkUtils.js
 @ ./node_modules/ask-sdk-core/dist/attributes/AttributesManagerFactory.js
 @ ./node_modules/ask-sdk-core/dist/index.js
 @ ./node_modules/ask-sdk/dist/index.js
 @ ./impfkompass.ts
</pre>

Running the output results in: 

<pre>
Error --------------------------------------------------
 
  Cannot find module "."
 
     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.
 
  Stack Trace --------------------------------------------
 
Error: Cannot find module "."
    at webpackMissingModule (/Users/johanneskees/Repositories/a7/impfkompass-alexa-node/.webpack/service/impfkompass.js:1768:67)
    at Object.createAskSdkUserAgent (/Users/johanneskees/Repositories/a7/impfkompass-alexa-node/.webpack/service/impfkompass.js:1768:145)
    at Skill.<anonymous> (/Users/johanneskees/Repositories/a7/impfkompass-alexa-node/.webpack/service/impfkompass.js:6632:58)
    at step (/Users/johanneskees/Repositories/a7/impfkompass-alexa-node/.webpack/service/impfkompass.js:6568:23)
    at Object.next (/Users/johanneskees/Repositories/a7/impfkompass-alexa-node/.webpack/service/impfkompass.js:6549:53)
    at fulfilled (/Users/johanneskees/Repositories/a7/impfkompass-alexa-node/.webpack/service/impfkompass.js:6540:58)
    at <anonymous>
...
</pre>
## Possible Solution
Remove usage of path.* in require calls

## Context

Webpack Conf: 
```
const path = require('path');
const slsw = require('serverless-webpack');

module.exports = {
    entry: slsw.lib.entries,
    devtool: 'source-map',
    resolve: {
        extensions: [
            '.js',
            '.jsx',
            '.json',
            '.ts',
            '.tsx'
        ]
    },
    output: {
        libraryTarget: 'commonjs',
        path: path.join(__dirname, '.webpack'),
        filename: '[name].js',
    },
    target: 'node',
    module: {
        loaders: [
            { test: /\.ts(x?)$/, loader: 'ts-loader' },
        ],
    },
};
```

TS Config:
```
{
  "compilerOptions": {
    "sourceMap": true,
    "lib": [
      "es5",
      "es2015.promise",
      "es6"
    ]
  }
}
```
## Your Environment

* ASK SDK for Node.js used: 2.0.1
* Node 10.0
* Webpack 3.11.0 + ts-loader 2.8.3
* Operating System and version:  Mac OS 10.3.5

**Comment by hoegertn** [comment]
Do you have an idea when this might be resolved? This blocks building skills with webpack using the new SDK. Is it really needed to add the version to the user-agent and therefore parse the `package.json`?

**Comment by dannolan** [comment]
I'm absolutely getting burned by this

**Comment by captnCC** [comment]
@eileennoonan83 This issue was fixed in v2.0.4, just update and you can continue using webpack!

**Comment by eileennoonan83** [comment]
@captnCC Thank you! Seems to be working now :D
